src/components/Sidebar/Schools.js

```diff
@@ -30,6 +30,7 @@ function Schools({ schools, isFetching, classes }) {
             <ListItem
               key={`${school.id}`}
               button
+              onClick={setBrowseLevel('subjects')}
```

Comment:
Oh, we forgot to finish talking about this! Touch base with me, we'll go over this.